Source/Painting/SvgGradientServer.cs

```diff
@@ -49,15 +41,15 @@ protected override void RemoveElement(SvgElement child)
         /// <summary>
         /// Gets the ramp of colors to use on a gradient.
         /// </summary>
-        public List<SvgGradientStop> Stops { get; private set; }
+        public List<SvgGradientStop> Stops { get; } = new List<SvgGradientStop>();
 
         /// <summary>
         /// Specifies what happens if the gradient starts or ends inside the bounds of the target rectangle.
         /// </summary>
         [SvgAttribute("spreadMethod")]
         public SvgGradientSpreadMethod SpreadMethod
         {
-            get { return GetAttribute("spreadMethod", false, SvgGradientSpreadMethod.Pad); }
+            get { return GetAttribute("spreadMethod", false, SvgDeferredPaintServer.TryGet<SvgGradientServer>(InheritGradient, null)?.SpreadMethod ?? SvgGradientSpreadMethod.Pad); }

```

Comment:
Considering performance, I want to be able to set method as default value, but I will not do it until drawing process is organized.